src/test/resources/readme.txt

```diff
@@ -0,0 +1,7 @@
+These chromedriver executables were downloaded from http://chromedriver.storage.googleapis.com/index.html
+
+Versions:
+  Win32:   2.14
+  linux64: 2.14
+  Mac32:   2.14
+  
```

Comment:
Would be sweet if you can remove the white spaces from the last line of the file